_locales/_experimental_en.json

```diff
@@ -69,18 +81,33 @@
     "message": "Remove account"
   },
   "a11y_goto_media_label": {
-    "message": "go to Gallery"
+    "message": "Go to gallery"
   },
   "a11y_goto_chat_label": {
-    "message": "go to Chat"
+    "message": "Go to chat"
   },
   "a11y_map_btn_label": {
-    "message": "map"
+    "message": "Map"
   },
   "a11y_main_menu_label": {
-    "message": "main menu"
+    "message": "Main menu"
   },
   "a11y_message_context_menu_btn_label": {
-    "message": "message actions"
+    "message": "Message actions"
+  },
+  "a11y_background_preview_label": {
+    "message": "Background preview"
+  },
+"welcome_desktop": {
+"message": "Welcome to Delta Chat"
```

Comment:
@OzancanKaratas Please fix indentation here. Thanks!

Reply:
Fixed in d3e4b56